[2025-02-06T15:24:07.926+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T15:24:07.952+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_physical_store_data scheduled__2025-02-06T00:01:00+00:00 [queued]>
[2025-02-06T15:24:07.963+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_physical_store_data scheduled__2025-02-06T00:01:00+00:00 [queued]>
[2025-02-06T15:24:07.964+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-06T15:24:07.988+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_physical_store_data> on 2025-02-06 00:01:00+00:00
[2025-02-06T15:24:08.006+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=255) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T15:24:08.007+0000] {standard_task_runner.py:72} INFO - Started process 260 to run task
[2025-02-06T15:24:08.008+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag_incremental', 'extract_physical_store_data', 'scheduled__2025-02-06T00:01:00+00:00', '--job-id', '314', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpci_s59qb']
[2025-02-06T15:24:08.010+0000] {standard_task_runner.py:105} INFO - Job 314: Subtask extract_physical_store_data
[2025-02-06T15:24:08.065+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_incremental.extract_physical_store_data scheduled__2025-02-06T00:01:00+00:00 [running]> on host f34e19f1f8c5
[2025-02-06T15:24:08.152+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag_incremental' AIRFLOW_CTX_TASK_ID='extract_physical_store_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T00:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-06T00:01:00+00:00'
[2025-02-06T15:24:08.153+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-06T15:24:08.153+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-06T15:24:08.153+0000] {logging_mixin.py:190} INFO - Current task name:extract_physical_store_data state:running start_date:2025-02-06 15:24:07.953624+00:00
[2025-02-06T15:24:08.153+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental and current dag run status:running
[2025-02-06T15:24:08.154+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T15:24:08.175+0000] {base.py:84} INFO - Retrieving connection 'physical_store_conn'
[2025-02-06T15:24:08.178+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM physical_orders 
            WHERE created_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-06T15:24:08.189+0000] {sql.py:562} INFO - Rows affected: 5076
[2025-02-06T15:24:08.275+0000] {etl.py:203} INFO - Extracted 5076 new rows from db_physical_store.
[2025-02-06T15:24:08.277+0000] {python.py:240} INFO - Done. Returned value was: /tmp/physical_data.json
[2025-02-06T15:24:08.300+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T15:24:08.301+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_incremental, task_id=extract_physical_store_data, run_id=scheduled__2025-02-06T00:01:00+00:00, execution_date=20250206T000100, start_date=20250206T152407, end_date=20250206T152408
[2025-02-06T15:24:08.309+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-06T15:24:08.309+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-06T15:24:08.309+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental queued_at:2025-02-06 15:24:02.325020+00:00
[2025-02-06T15:24:08.309+0000] {logging_mixin.py:190} INFO - Task hostname:f34e19f1f8c5 operator:PythonOperator
[2025-02-06T15:24:08.342+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T15:24:08.365+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-06T15:24:08.366+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-07T11:21:20.346+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-07T11:21:20.371+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_physical_store_data scheduled__2025-02-06T00:01:00+00:00 [queued]>
[2025-02-07T11:21:20.386+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_physical_store_data scheduled__2025-02-06T00:01:00+00:00 [queued]>
[2025-02-07T11:21:20.387+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-07T11:21:20.401+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_physical_store_data> on 2025-02-06 00:01:00+00:00
[2025-02-07T11:21:20.422+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=96) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-07T11:21:20.426+0000] {standard_task_runner.py:72} INFO - Started process 102 to run task
[2025-02-07T11:21:20.425+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag_incremental', 'extract_physical_store_data', 'scheduled__2025-02-06T00:01:00+00:00', '--job-id', '612', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpw91bh1s2']
[2025-02-07T11:21:20.427+0000] {standard_task_runner.py:105} INFO - Job 612: Subtask extract_physical_store_data
[2025-02-07T11:21:20.504+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_incremental.extract_physical_store_data scheduled__2025-02-06T00:01:00+00:00 [running]> on host f34e19f1f8c5
[2025-02-07T11:21:20.636+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag_incremental' AIRFLOW_CTX_TASK_ID='extract_physical_store_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T00:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-06T00:01:00+00:00'
[2025-02-07T11:21:20.638+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-07T11:21:20.639+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-07T11:21:20.640+0000] {logging_mixin.py:190} INFO - Current task name:extract_physical_store_data state:running start_date:2025-02-07 11:21:20.372273+00:00
[2025-02-07T11:21:20.641+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental and current dag run status:running
[2025-02-07T11:21:20.641+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-07T11:21:20.665+0000] {base.py:84} INFO - Retrieving connection 'physical_store_conn'
[2025-02-07T11:21:20.668+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM physical_orders 
            WHERE created_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-07T11:21:20.693+0000] {sql.py:562} INFO - Rows affected: 9478
[2025-02-07T11:21:20.982+0000] {etl.py:203} INFO - Extracted 9478 new rows from db_physical_store.
[2025-02-07T11:21:20.988+0000] {python.py:240} INFO - Done. Returned value was: /tmp/physical_data.json
[2025-02-07T11:21:21.023+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-07T11:21:21.024+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_incremental, task_id=extract_physical_store_data, run_id=scheduled__2025-02-06T00:01:00+00:00, execution_date=20250206T000100, start_date=20250207T112120, end_date=20250207T112121
[2025-02-07T11:21:21.041+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-07T11:21:21.042+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-07T11:21:21.042+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental queued_at:2025-02-07 11:21:14.687092+00:00
[2025-02-07T11:21:21.043+0000] {logging_mixin.py:190} INFO - Task hostname:f34e19f1f8c5 operator:PythonOperator
[2025-02-07T11:21:21.087+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-07T11:21:21.143+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-07T11:21:21.144+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
